src/core/cuda_wrapper.hpp

```diff
@@ -25,7 +25,6 @@
 #define cudaFreeHost hipHostFree
 #define cudaHostAlloc hipHostMalloc
 #define cudaHostAllocWriteCombined hipHostMallocWriteCombined
-#define cudaLaunchKernelGGL hipLaunchKernelGGL
```

Comment:
Is that intentional?

Reply:
Since the CI passes (apart from an unrelated GitLab CI Runner error), this function is probably unused an can be safely removed.